kafka-init/create_topics.py

```diff
@@ -96,7 +96,7 @@ def list_topics():
 
 def create_topic(name, partitions, replicas, configs=None):
     if configs:
-        arg_pairs = map(lambda item: ['--config', '%s=%s' % item],
+        arg_pairs = map(lambda item: ['--config', '{0}={0}'.format(item)],
```

Comment:
`item` is a tuple with 2 values, '%' expands this but `.format()` will not... we don't want `'{0}={0}'.format(item)` but rather `'{0}={1}'.format(*item)`

(though to be honest I'm not sure of the utility of converting all `%` formatting to use `.format()` anyway...)